compiler/ccgexprs.nim

```diff
@@ -3329,6 +3333,8 @@ proc genConstObjConstr(p: BProc; n: PNode; isConst: bool; result: var Rope) =
 
 proc genConstSimpleList(p: BProc, n: PNode; isConst: bool; result: var Rope) =
   result.add "{"
+  if p.vccAndC and n.len == 0:
+    result.add "0"
```

Comment:
How do you know here that `0` is a valid element for all the constructs that happen to use `genConstSimpleList`?

Reply:
how to simulate empty varargs?  seems it never be.